src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Children.cs

```diff
@@ -18,20 +18,20 @@ public static partial class IViewModelExtensions
 		/// Gets the children of a <see cref="IViewModel"/>.
 		/// </summary>
 		/// <param name="viewModel">This <see cref="IViewModel"/>.</param>
-		/// <returns>The children <see cref="IViewModel"/>s of this viewmodel.</returns>
+		/// <returns>A IEnumerable of <see cref="IViewModel"/>s registered to this ViewModel.</returns>
```

Comment:
I wouldn't introduce the term "registered". Keep using the "children" term. You can explain what being a child means in the summary.